org-super-agenda.el

```diff
@@ -1099,6 +1114,53 @@ actually the ORDER for the groups."
        (-remove #'s-blank-str? it)
        (s-join "\n" it)))
 
+;;;; Hide/Show filter
+(defun org-super-agenda--hide-or-show-groups (&rest _)
+  "Hide/Show any empty/non-empty groups after `org-agenda-finalize',
+`org-agenda-filter-apply'  was called."
+  (cl-labels ((header-p () (org-get-at-bol 'org-super-agenda-header))
+              (grid-p () (not (or  (org-get-at-bol 'org-agenda-structural-header)
+                                   (org-get-at-bol 'org-agenda-date-header)
+                                   (org-get-at-bol 'type)
+                                   (org-get-at-bol 'org-super-agenda-header))))
+              (group-item-visible-p () (and (org-get-at-bol 'type) (not (org-get-at-bol 'invisible))))
+              (next-header () (let (header nohide grid-end)
+                                (while (not (or (bobp) header))
+                                  (cond
```

Comment:
Nitpick: It's a matter of style, but for consistency with the rest of the code, please put the first cond on the same line as `cond`.